src/index.html

```diff
@@ -7,8 +7,77 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      type="text/css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home" class="nav__img"><img src="images/logo.png" alt="logo"></a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >apple</a>
```

Comment:
```suggestion
            <a
              href="#apple"
              class="nav__link is-active"
            >
              apple
            </a>
```
Or you can make it in one line if there are < 3 attributes:
`<a href="#apple" class="nav__link is-active">apple</a>`